docs/delegate-resources/rubin-data-academy-2024.rst

```diff
@@ -0,0 +1,158 @@
+	- If a section within the template is not needed, comment out the section title and label reference. Do not delete the expected section title, reference or related comments provided from the template.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+.. This is the label that can be used for cross referencing this file.
+.. Recommended title label format is "Directory Name"-"Title Name" -- Spaces should be replaced by hyphens.
+.. _DP0-Delegate-Resources-RDA2024:
+.. Each section should include a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link and explicit title using the syntax :ref:`link text <label-name>`.
+.. A warning will alert you of identical labels during the linkcheck process.
+
+.. This section should provide a brief, top-level description of the page.
+
+
+
+.. _DP0-Delegate-Resources-RDA2024-overview:
+
+========
+Overview
+========
+
+**Dates**: June 17-18, 20-21, 2024
+
+**Times** (delegates may attend either session):
+
+Session A: 8:00-11:00 PDT (15:00-18:00 UTC)
+
+Session B: 19:00-21:00 PDT (02:00-04:00 UTC; +1 day)
+
+**Location**: virtual
+
+**Eligibility**: All DP0 delegates are eligible to register for the DP0 Rubin Data Academy.
+There is currently space available for new DP0 delegates:
+for more information about how to become a delegate, see the "Getting started" `document <https://dp0.lsst.io/delegate-resources/getting-started.html>`_.
+
+**What is DP0?**
+Rubin Data Previews use simulated data products, but the tools and the data formats are similar to what the real data and the `Rubin Science Platform (RSP) <https://dp0-2.lsst.io/data-access-analysis-tools/index.html#rubin-science-platform-rsp>`_ will look like.
+This provides an excellent opportunity to prepare for the release of the first real data products.
+There are two distinct datasets that are available for the Delegates in the Data Preview 0 (DP0).
+
+**Data Preview 0.2** (`DP0.2 <https://dp0-2.lsst.io>`_), which was released on June 30, 2022, is based on simulated
+Legacy Survey of Space and Time (LSST)-like images that were generated by the Dark Energy Science Collaboration (DESC),
+and then processed with the LSST Science Pipelines to provide data products (both images and catalogs)
+in the same format as future (real) data releases.  These data products can be accessed and analyzed
+via the web-based Rubin Science Platform (RSP).
+
+**Data Preview 0.3** (`DP0.3 <https://dp0-3.lsst.io/index.html>`_)
+which was released around July 1 2023, contains only
```

Comment:
removed "which was" and add a comma after July 1, 2023

Reply:
add comma after Data Preview 0.3

Reply:
done